test/uk/gov/hmrc/helptosave/controllers/EnrolmentStoreControllerSpec.scala

```diff
@@ -133,21 +138,53 @@ class EnrolmentStoreControllerSpec extends AuthSupport with GeneratorDrivenPrope
 
         m.foreach{
           case (s, j) ⇒
-            mockAuth(AuthWithCL200, v2Nino)(Right(mockedNinoRetrieval))
+            mockAuth(GGAndPrivilegedProviders, authProviderId)(Right(ggCredentials))
+            mockAuth(v2Nino)(Right(mockedNinoRetrieval))
             mockEnrolmentStoreGet(nino)(Right(s))
 
-            val result = getEnrolmentStatus()
+            val result = getEnrolmentStatus(Some(nino))
             status(result) shouldBe OK
             contentAsJson(result) shouldBe Json.parse(j)
         }
       }
 
       "return an error if the call was not successful" in {
-        mockAuth(AuthWithCL200, v2Nino)(Right(mockedNinoRetrieval))
+        mockAuth(GGAndPrivilegedProviders, authProviderId)(Right(ggCredentials))
+        mockAuth(v2Nino)(Right(mockedNinoRetrieval))
         mockEnrolmentStoreGet(nino)(Left(""))
 
-        status(getEnrolmentStatus()) shouldBe INTERNAL_SERVER_ERROR
+        status(getEnrolmentStatus(Some(nino))) shouldBe INTERNAL_SERVER_ERROR
+      }
+    }
+
+    "handling requests to get enrolment status via calls from Stride" must {
```

Comment:
these could be privileged API calls as well - maybe it's more general to say something like "handling requests with privileged access"? There's a similar sentence in the eligibility check unit tests

Reply:
renamed